main.go

```diff
@@ -72,10 +149,27 @@ func coreMessageHandler(session *discordgo.Session, message *discordgo.MessageCr
 		// It's not a command, nothing to do here.
 		return
 	}
+}
+
+func commandHandler(discord *discordgo.Session, message *discordgo.MessageCreate) {
+	user := message.Author
+	if user.ID == botID || user.Bot {
+		return
+	}
 
 	for _, handler := range data.Globals().CommandHandlers {
-		handler.(func(*discordgo.Session, *discordgo.MessageCreate))(session, message)
+		handler.(func(*discordgo.Session, *discordgo.MessageCreate))(discord, message)
 	}
 
 	fmt.Printf("Message: %+v || From: %s\n", message.Message, message.Author)
+
+	if strings.HasPrefix(message.Message.Content, "!lfg ") {
+		fmt.Printf("LFG Command string found.\n")
+		specialCharsRegex, err := regexp.Compile("[^a-zA-Z0-9]+")
+		if err != nil {
+			discord.MessageReactionAdd(message.Message.ChannelID, message.Message.ID, "632789111958274068")
+		}
+		gameSuggestion := strings.ToLower(specialCharsRegex.ReplaceAllString(strings.TrimPrefix(message.Message.Content, "!lfg "), ""))
+		discord.ChannelMessageSend(message.Message.ChannelID, fmt.Sprintf("%s has been added to the LFG queue for %s for the next 4 hours.", message.Message.Author.Username, gameSuggestion))
```

Comment:
This should be built into a separate command function (see /commands/chapter.go:PartyOnCommandHandler() for an example).